Image Created in JIB mode for spring-boot-camel-complete Quickstart has wrong permissions

### Description
<!--
Thank you for trying out Eclipse JKube!

Please fill out the following information to help us in analyzing the issue, but feel free to skip it if you don't have the information at hand or if it does not apply. Please remove every section which does not apply to the issue.
-->
I encountered this issue while working on #903 , When I try building image in Docker mode and run it; everything seems to be working okay:
```
spring-boot-camel-complete : $ docker run 26fe2
Starting the Java application using /opt/jboss/container/java/run/run-java.sh ...
INFO exec  java -javaagent:/usr/share/java/jolokia-jvm-agent/jolokia-jvm.jar=config=/opt/jboss/container/jolokia/etc/jolokia.properties -javaagent:/usr/share/java/prometheus-jmx-exporter/jmx_prometheus_javaagent.jar=9779:/opt/jboss/container/prometheus/etc/jmx-exporter-config.yaml -XX:+UseParallelOldGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/spring-boot-camel-complete-1.5.0-SNAPSHOT.jar  
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jolokia.util.ClassUtil (file:/usr/share/java/jolokia-jvm-agent/jolokia-jvm.jar) to constructor sun.security.x509.X500Name(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.jolokia.util.ClassUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
I> No access restrictor found, access to any MBean is allowed
Jolokia: Agent started with URL https://172.17.0.2:8778/jolokia/

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.4)

2021-09-23 11:55:55.582  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : Starting Application using Java 11.0.12 on 82fab4560ebf with PID 1 (/deployments/spring-boot-camel-complete-1.5.0-SNAPSHOT.jar started by jboss in /deployments)
2021-09-23 11:55:55.584  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : No active profile set, falling back to default profiles: default
2021-09-23 11:55:56.992  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2021-09-23 11:55:57.004  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2021-09-23 11:55:57.004  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.52]
2021-09-23 11:55:57.073  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2021-09-23 11:55:57.074  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1426 ms
2021-09-23 11:55:57.965  INFO 1 --- [           main] o.a.c.c.s.CamelHttpTransportServlet      : Initialized CamelHttpTransportServlet[name=CamelServlet, contextPath=]
2021-09-23 11:55:57.967  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2021-09-23 11:55:58.245  INFO 1 --- [           main] c.s.b.CamelSpringBootApplicationListener : Starting CamelMainRunController to ensure the main thread keeps running
2021-09-23 11:55:58.258  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   : Routes startup summary (total:5 started:5)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route1 (timer://order)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route2 (file://work/orders/input)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route3 (timer://foo)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route4 (direct://hello-camel)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route5 (rest://get:/hello-camel)
2021-09-23 11:55:58.259  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   : Apache Camel 3.11.1 (camel-1) started in 291ms (build:63ms init:209ms start:19ms)
2021-09-23 11:55:58.264  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : Started Application in 3.274 seconds (JVM running for 4.03)
2021-09-23 11:55:59.269  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 11:55:59.271  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 44E8F91177495F2-0000000000000000
2021-09-23 11:55:59.280  INFO 1 --- [- timer://order] route1                                   : Generating order order1.xml
2021-09-23 11:55:59.768  INFO 1 --- [rk/orders/input] route2                                   : Processing order order1.xml
2021-09-23 11:56:00.258  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 11:56:00.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 44E8F91177495F2-0000000000000003
2021-09-23 11:56:01.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 11:56:01.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 44E8F91177495F2-0000000000000004
2021-09-23 11:56:02.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 11:56:02.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 44E8F91177495F2-0000000000000005
2021-09-23 11:56:03.259  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 11:56:03.260  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 44E8F91177495F2-0000000000000006
```

However, when I build same project with JIB build strategy; I see this strange error:
```
spring-boot-camel-complete : $ docker run 515094d11ac7
Starting the Java application using /opt/jboss/container/java/run/run-java.sh ...
INFO exec  java -javaagent:/usr/share/java/jolokia-jvm-agent/jolokia-jvm.jar=config=/opt/jboss/container/jolokia/etc/jolokia.properties -javaagent:/usr/share/java/prometheus-jmx-exporter/jmx_prometheus_javaagent.jar=9779:/opt/jboss/container/prometheus/etc/jmx-exporter-config.yaml -XX:+UseParallelOldGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/spring-boot-camel-complete-1.5.0-SNAPSHOT.jar  
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jolokia.util.ClassUtil (file:/usr/share/java/jolokia-jvm-agent/jolokia-jvm.jar) to constructor sun.security.x509.X500Name(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.jolokia.util.ClassUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
I> No access restrictor found, access to any MBean is allowed
Jolokia: Agent started with URL https://172.17.0.2:8778/jolokia/

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.4)

2021-09-23 12:00:02.031  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : Starting Application using Java 11.0.12 on a3bee18801cf with PID 1 (/deployments/spring-boot-camel-complete-1.5.0-SNAPSHOT.jar started by jboss in /deployments)
2021-09-23 12:00:02.035  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : No active profile set, falling back to default profiles: default
2021-09-23 12:00:03.170  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2021-09-23 12:00:03.180  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2021-09-23 12:00:03.180  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.52]
2021-09-23 12:00:03.244  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2021-09-23 12:00:03.245  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1131 ms
2021-09-23 12:00:03.903  INFO 1 --- [           main] o.a.c.c.s.CamelHttpTransportServlet      : Initialized CamelHttpTransportServlet[name=CamelServlet, contextPath=]
2021-09-23 12:00:03.905  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2021-09-23 12:00:04.130  WARN 1 --- [           main] o.a.camel.component.file.FileEndpoint    : Cannot auto create starting directory: work/orders/input
2021-09-23 12:00:04.152  INFO 1 --- [           main] c.s.b.CamelSpringBootApplicationListener : Starting CamelMainRunController to ensure the main thread keeps running
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   : Routes startup summary (total:5 started:5)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route1 (timer://order)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route2 (file://work/orders/input)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route3 (timer://foo)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route4 (direct://hello-camel)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   :     Started route5 (rest://get:/hello-camel)
2021-09-23 12:00:04.163  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   : Apache Camel 3.11.1 (camel-1) started in 236ms (build:37ms init:184ms start:15ms)
2021-09-23 12:00:04.167  INFO 1 --- [           main] o.e.j.q.g.s.Application                  : Started Application in 2.85 seconds (JVM running for 3.509)
2021-09-23 12:00:05.164  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : Hello from Camel!
2021-09-23 12:00:05.165  INFO 1 --- [2 - timer://foo] o.e.j.q.g.s.CamelRouteBuilder            : My id is 6EC7FE49F07E9D1-0000000000000000
2021-09-23 12:00:05.168  INFO 1 --- [- timer://order] route1                                   : Generating order order1.xml
2021-09-23 12:00:05.174 ERROR 1 --- [- timer://order] o.a.c.p.e.DefaultErrorHandler            : Failed delivery for (MessageId: 6EC7FE49F07E9D1-0000000000000001 on ExchangeId: 6EC7FE49F07E9D1-0000000000000001). Exhausted after delivery attempt: 1 caught: org.apache.camel.component.file.GenericFileOperationFailedException: Cannot store file: work/orders/input/order1.xml

Message History (complete message history is disabled)
---------------------------------------------------------------------------------------------------------------------------------------
RouteId              ProcessorId          Processor                                                                        Elapsed (ms)
[route1            ] [route1            ] [from[timer://order?period=5000]                                               ] [         8]
	...
[route1            ] [to1               ] [file:work/orders/input                                                        ] [         0]

Stacktrace
---------------------------------------------------------------------------------------------------------------------------------------


org.apache.camel.component.file.GenericFileOperationFailedException: Cannot store file: work/orders/input/order1.xml
	at org.apache.camel.component.file.FileOperations.storeFile(FileOperations.java:406) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.writeFile(GenericFileProducer.java:290) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.processExchange(GenericFileProducer.java:173) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.process(GenericFileProducer.java:78) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66) ~[camel-support-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.SendProcessor.process(SendProcessor.java:172) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$SimpleTask.run(RedeliveryErrorHandler.java:463) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.DefaultReactiveExecutor$Worker.schedule(DefaultReactiveExecutor.java:179) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.DefaultReactiveExecutor.scheduleMain(DefaultReactiveExecutor.java:64) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.Pipeline.process(Pipeline.java:184) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.CamelInternalProcessor.process(CamelInternalProcessor.java:398) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.timer.TimerConsumer.sendTimerExchange(TimerConsumer.java:210) ~[camel-timer-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.timer.TimerConsumer$1.run(TimerConsumer.java:76) ~[camel-timer-3.11.1.jar!/:3.11.1]
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556) ~[na:na]
	at java.base/java.util.TimerThread.run(Timer.java:506) ~[na:na]
Caused by: java.nio.file.NoSuchFileException: work/orders/input/order1.xml
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[na:na]
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[na:na]
	at java.base/java.nio.file.Files.newByteChannel(Files.java:371) ~[na:na]
	at java.base/java.nio.file.Files.newByteChannel(Files.java:422) ~[na:na]
	at org.apache.camel.component.file.FileOperations.prepareOutputFileChannel(FileOperations.java:531) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.FileOperations.writeFileByStream(FileOperations.java:455) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.FileOperations.storeFile(FileOperations.java:388) ~[camel-file-3.11.1.jar!/:3.11.1]
	... 14 common frames omitted

2021-09-23 12:00:05.176  WARN 1 --- [- timer://order] o.a.camel.component.timer.TimerConsumer  : Error processing exchange. Exchange[6EC7FE49F07E9D1-0000000000000001]. Caused by: [org.apache.camel.component.file.GenericFileOperationFailedException - Cannot store file: work/orders/input/order1.xml]

org.apache.camel.component.file.GenericFileOperationFailedException: Cannot store file: work/orders/input/order1.xml
	at org.apache.camel.component.file.FileOperations.storeFile(FileOperations.java:406) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.writeFile(GenericFileProducer.java:290) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.processExchange(GenericFileProducer.java:173) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.GenericFileProducer.process(GenericFileProducer.java:78) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66) ~[camel-support-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.SendProcessor.process(SendProcessor.java:172) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$SimpleTask.run(RedeliveryErrorHandler.java:463) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.DefaultReactiveExecutor$Worker.schedule(DefaultReactiveExecutor.java:179) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.DefaultReactiveExecutor.scheduleMain(DefaultReactiveExecutor.java:64) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.processor.Pipeline.process(Pipeline.java:184) ~[camel-core-processor-3.11.1.jar!/:3.11.1]
	at org.apache.camel.impl.engine.CamelInternalProcessor.process(CamelInternalProcessor.java:398) ~[camel-base-engine-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.timer.TimerConsumer.sendTimerExchange(TimerConsumer.java:210) ~[camel-timer-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.timer.TimerConsumer$1.run(TimerConsumer.java:76) ~[camel-timer-3.11.1.jar!/:3.11.1]
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556) ~[na:na]
	at java.base/java.util.TimerThread.run(Timer.java:506) ~[na:na]
Caused by: java.nio.file.NoSuchFileException: work/orders/input/order1.xml
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[na:na]
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[na:na]
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[na:na]
	at java.base/java.nio.file.Files.newByteChannel(Files.java:371) ~[na:na]
	at java.base/java.nio.file.Files.newByteChannel(Files.java:422) ~[na:na]
	at org.apache.camel.component.file.FileOperations.prepareOutputFileChannel(FileOperations.java:531) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.FileOperations.writeFileByStream(FileOperations.java:455) ~[camel-file-3.11.1.jar!/:3.11.1]
	at org.apache.camel.component.file.FileOperations.storeFile(FileOperations.java:388) ~[camel-file-3.11.1.jar!/:3.11.1]
	... 14 common frames omitted
```

On looking closer it looks like in case of image created with JIB mode, there isn't permission to create `work` directory:
```
spring-boot-camel-complete : $ docker run -it 7a48d09f52b6 /bin/bash
id: cannot find name for user ID 1000
[I have no name!@36e0a88cfa0b ~]$ cd /deployments
[I have no name!@36e0a88cfa0b deployments]$ ls
data  spring-boot-camel-complete-1.4.0.jar
[I have no name!@36e0a88cfa0b deployments]$ mkdir work
mkdir: cannot create directory ‘work’: Permission denied
[I have no name!@36e0a88cfa0b deployments]$ ls
```

I see the same behavior in both maven and gradle plugins. We need to figure out what's different when we're creating image using JIB and apply appropriate fix.
### Info

* Eclipse JKube version : v1.4.0

* If it's a bug, how to reproduce : Go to `spring-boot-camel-complete` quickstart and try running image build in Docker Mode and JIB mode

--- a/quickstarts/gradle/spring-boot-camel-complete/src/main/java/org/eclipse/jkube/quickstarts/gradle/springbootcamelcomplete/CamelRouteBuilder.java
+++ b/quickstarts/gradle/spring-boot-camel-complete/src/main/java/org/eclipse/jkube/quickstarts/gradle/springbootcamelcomplete/CamelRouteBuilder.java
@@ -44,11 +44,11 @@ public class CamelRouteBuilder extends RouteBuilder {
         .setHeader(Exchange.FILE_NAME)
         .method(OrderGenerator.class, "generateFileName")
         .log("Generating order ${file:name}")
-        .to("file:work/orders/input");
+        .to("file:/tmp/work/orders/input");
     // Route to consume these orders and move them to a processed directory
-    from("file:work/orders/input")
+    from("file:/tmp/work/orders/input")
         .log("Processing order ${file:name}")
-        .to("file:work/orders/processed");
+        .to("file:/tmp/work/orders/processed");
     // Route to log a Hello world message every second
     from("timer:foo?period=1s")
         .setBody().constant(camelGreeting)

--- a/quickstarts/maven/spring-boot-camel-complete/src/main/java/org/eclipse/jkube/quickstart/springboot/camel/log/CamelRouteBuilder.java
+++ b/quickstarts/maven/spring-boot-camel-complete/src/main/java/org/eclipse/jkube/quickstart/springboot/camel/log/CamelRouteBuilder.java
@@ -44,11 +44,11 @@ public class CamelRouteBuilder extends RouteBuilder {
         .setHeader(Exchange.FILE_NAME)
         .method(OrderGenerator.class, "generateFileName")
         .log("Generating order ${file:name}")
-        .to("file:work/orders/input");
+        .to("file:/tmp/work/orders/input");
     // Route to consume these orders and move them to a processed directory
-    from("file:work/orders/input")
+    from("file:/tmp/work/orders/input")
         .log("Processing order ${file:name}")
-        .to("file:work/orders/processed");
+        .to("file:/tmp/work/orders/processed");
     // Route to log a Hello world message every second
     from("timer:foo?period=1s")
         .setBody().constant(camelGreeting)

